package.json

```diff
@@ -76,6 +76,11 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12.1.5",
     "@testing-library/user-event": "^14.4.3",
+    "@types/jest": "^28.1.8",
```

Comment:
Are these here now because we need different versions for development vs. production?

Reply:
Oh never mind, these are type-script specific libraries.

Reply:
The `@types/...` packages contain type definitions for packages that don't provide definitions themselves. They're not separate versions of those libraries.

https://github.com/DefinitelyTyped/DefinitelyTyped